acquisitions/src/main/resources/thunderjet/consortia/consortia-orders-junit.feature

```diff
@@ -0,0 +1,160 @@
+    * def patronUser = { id: '#(patronUserId)', username: 'college_patron_user', password: 'college_patron_user_password', type: 'patron', tenant: '#(collegeTenant)'}
+
+    * def userToUpdate = { id: '#(userToUpdateId)', username: 'user_to_update', password: 'user_to_update_password', type: 'staff', tenant: '#(centralTenant)', phone: '#(userPhone)', mobilePhone: '#(userMobilePhone)'}
+    * def universityUserToUpdate = { id: '#(universityUserToUpdateId)', username: 'university_user_to_update', password: 'university_user_to_update_password', type: 'staff', tenant: '#(universityTenant)', phone: '#(userPhone)', mobilePhone: '#(userMobilePhone)'}
+    * def patronUserToUpdate = { id: '#(patronUserToUpdateId)', username: 'patron_user_to_update', password: 'patron_user_to_update_password', type: 'patron', tenant: '#(collegeTenant)', phone: '#(userPhone)', mobilePhone: '#(userMobilePhone)'}
+
+    # define custom login
+    * def login = read('classpath:common-consortia/initData.feature@Login')
+
+  Scenario: Create ['central', 'university', 'college'] tenants and set up admins
+    * call read('classpath:common-consortia/tenant-and-local-admin-setup.feature@SetupTenant') { tenant: '#(centralTenant)', admin: '#(consortiaAdmin)'}
+    * call read('classpath:common-consortia/tenant-and-local-admin-setup.feature@SetupTenant') { tenant: '#(universityTenant)', admin: '#(universityUser1)'}
+    * call read('classpath:common-consortia/tenant-and-local-admin-setup.feature@SetupTenant') { tenant: '#(collegeTenant)', admin: '#(collegeUser1)'}
+
+    # create users in all tenants
+    * call read('classpath:common-consortia/create-users.feature@CreateUsers')
+
+    # add 'consortia.all' (for consortia management) and 'tags.all' (for publish coordinator tests) permissions to main users
+    * call login consortiaAdmin
+    * call read('classpath:common-consortia/initData.feature@PutPermissions') { desiredPermissions: ['consortia.all', 'tags.all']}
+
+    * call login universityUser1
+    * call read('classpath:common-consortia/initData.feature@PutPermissions') { desiredPermissions: ['consortia.all', 'tags.all']}
+
+    * call login collegeUser1
+    * call read('classpath:common-consortia/initData.feature@PutPermissions') { desiredPermissions: ['consortia.all', 'tags.all']}
+
+  Scenario: Consortium api tests
+    * call read('tenant-utils/consortium.feature')
+
+  Scenario: Tenant api tests
+    * call read('tenant-utils/tenant.feature')
+
+  Scenario: verify and setup 'consortiaAdmin' for all tenants
+    * call read('tenant-utils/consortia-admin-verification-and-setup.feature')
+
+  Scenario: verify 'consortia-system-user' in all tenants
+    * call read('tenant-utils/consortia-system-users-verification.feature')
+
+  Scenario: User-Tenant associations api tests
+    * call read('tenant-utils/user-tenant-associations.feature')
```

Comment:
can you check please if test passes without 

- verify 'consortia-system-user' in all tenants

- User-Tenant associations api tests

Reply:
if our orders test passes without them we also should delete lot of variables that is not used lilke universityUser1Id, shadowUser, ...

Reply:
removed both of them